components/suggestion/search_channel_provider.jsx

```diff
@@ -26,12 +26,31 @@ function itemToName(item) {
 }
 
 class SearchChannelSuggestion extends Suggestion {
+    componentDidMount() {
```

Comment:
Instead of implementing this in both the `SearchChannelSuggestion` and `SearchUserSuggestion` components, couldn't you just make the `SearchSuggestionList` handle this itself? It already knows what value has been suggested, and we won't have to use IDs to do this then

Reply:
Done @hmhealey